index.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
```

Comment:
I don't think this file is needed anymore. It looks like home.html is your main page!